solidity/contracts/PhasedEscrow.sol

```diff
@@ -71,3 +71,82 @@ contract CurveRewardsEscrowBeneficiary is Ownable {
         curveRewards.notifyRewardAmount(amount);
     }
 }
+
+/// @dev Interface of recipient contract for approveAndCall pattern.
+interface IStakerRewards { function receiveApproval(address _from, uint256 _value, address _token, bytes calldata _extraData) external; }
```

Comment:
Let's apply some nicer formatting to this interface.

Reply:
Done.